docs/data/base/components/number-field/number-field.md

```diff
@@ -157,11 +157,11 @@ The `format` prop accepts [`Intl.NumberFormat` options](https://developer.mozill
 
 ## Scrubbing
 
-The `ScrubArea` subcomponent lets users increment/decrement the value via a click+drag interaction with pointer, as a faster alternative to the stepper buttons. This is useful in high-density UIs, such as an image editor that changes the width, height, or location of a layer. You could wrap an icon or a `<label/>` in the `ScrubArea` component.
+The `<ScrubArea />` subcomponent lets users increment/decrement the value via a click+drag interaction with pointer, as a faster alternative to the stepper buttons. This is useful in high-density UIs, such as an image editor that changes the width, height, or location of a layer. You could wrap an icon or a `<label/>` in the `<ScrubArea />` component.
```

Comment:
I'm not a fan of adding brackets whenever a component name appears (especially that technically this is incorrect - for example, NumberInput.Root won't ever be self-closing as it requires children to work correctly).

I'd go with either `NumberInput.ScrubArea` (with backticks) when we use the namespace or just ScrubArea in plain text.